src/v/storage/tests/storage_e2e_test.cc

```diff
@@ -1955,7 +1994,11 @@ FIXTURE_TEST(changing_cleanup_policy_back_and_forth, storage_test_fixture) {
     BOOST_REQUIRE_EQUAL(disk_log->segment_count(), 3);
 
     storage::compaction_config c_cfg(
-      model::timestamp::min(), std::nullopt, ss::default_priority_class(), as);
+      model::timestamp::min(),
+      std::nullopt,
+      model::offset::max(),
+      ss::default_priority_class(),
+      as);
```

Comment:
it looks like further down in the file there are other compaction configs that need updated too?

Reply:
I believe I misplaced one chunk during one of the rebases. I'm making sure this config change can be bisected now, thanks.